src/main/resources/assets/js/components/AppListComponent.jsx

```diff
@@ -99,6 +100,11 @@ var AppListComponent = React.createClass({
               </span>
             </th>
             <th className="text-right">
+              <span className={headerClassSet}>
```

Comment:
I've put them in the order of what I deemed to be the highest importance, but feel free to elaborate on if something else makes more sense?